test/node.spec.js

```diff
@@ -0,0 +1,56 @@
+const { spawnSync } = require("child_process");
+const nodes = require("../src/nodes");
+
+const expectedUnhandledNodes = [
+  "arg_ambiguous",
+  "args_add",
+  "args_new",
+  "heredoc_dedent",
+  "magic_comment",
+  "mlhs_add",
+  "mlhs_new",
+  "mrhs_add",
+  "mrhs_new",
+  "operator_ambiguous",
+  "parse_error",
+  "qsymbols_add",
+  "qsymbols_new",
+  "qwords_add",
+  "qwords_new",
+  "regexp_add",
+  "regexp_new",
+  "stmts_add",
+  "stmts_new",
+  "string_add",
+  "string_content",
+  "symbols_add",
+  "symbols_new",
+  "void_stmt",
+  "words_add",
+  "words_new",
+  "xstring_add",
+  "xstring_new",
+  ""
+];
+
+const possibleNodes = function() {
+  const child = spawnSync("ruby", [
+    "-rripper",
+    "-e",
+    "puts Ripper::PARSER_EVENTS"
+  ]);
+
+  const error = child.stderr.toString();
+  if (error) {
+    throw new Error(error);
+  }
+
+  return child.stdout.toString().split("\n");
+};
+
+describe("node support", () => {
+  test.skip("handles all ripper parsing events", () => {
```

Comment:
From what I can tell, these tests weren't running on master. I tidied it up a bit, but it'll require actual semantic changes to get them working again. Happy to implement the changes to get them passing if it makes sense as part of this PR. I just wanted to call this out explicitly as the PR was purely meant to be "run the tests in parallel" - and avoiding changing the semantics of any tests.